### DISCUSSION TITLE
Audio and IF Filters for reducing the background noise

### DISCUSSION THREAD

**Discussion by ja-Bertolin** [AlexandreRouma/SDRPlusPlus#654] [Ideas]
It would be great whether it could be added Audio/IF Filters in order to improve audio quality and reduce the background noise.

Thanks

**Comment by the-gobleo-tinkerer** [comment]
Please see #183 and #544 as this is a duplicate question. Currently there is only an IF noise reduction option for FM (both wide and narrow).
